src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  @for $i from 0 through 5 {
+    &--#{$i} > &__star:nth-child(-n + #{$i}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
The loop starts from 0, which means you are creating a modifier class `.stars--0` that doesn't change any stars. It would be more efficient to start the loop from 1 since having 0 active stars does not require any additional styles. You can adjust the loop to start from 1 to avoid generating unnecessary CSS.